src/galaxy/__init__.py

```diff
@@ -5,7 +5,10 @@
 import logging
 import os
 
-def get_db_connection_params() -> dict:
```

Comment:
we should put this function in config.py instead of init , so that app.py can access easily